examples/green_screen/main.cpp

```diff
@@ -0,0 +1,490 @@
+#include <algorithm>
+#include <iostream>
+#include <vector>
+#include <string>
+#include <chrono>
+#include <getopt.h>
+
+#include <k4a/k4a.hpp>
+#include <opencv2/calib3d.hpp>
+#include <opencv2/core.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+
+using std::cout;
+using std::endl;
+using std::vector;
+
+#include "transformation.h"
+#include "MultiDeviceCapturer.h"
+
+// Allowing at least 160 microseconds between depth cameras should ensure they do not interfere with one another.
+constexpr uint32_t MIN_TIME_BETWEEN_DEPTH_CAMERA_PICTURES_USEC = 160;
+
+// ideally, we could generalize this to many OpenCV types
+cv::Mat color_to_opencv(const k4a::image &im)
+{
+    // TODO this only handles mjpg
```

Comment:
Is this a note or a todo? Do you plan to handle other types?

Reply:
I'd wanted to see how difficult it would be- if it's simple, then it'd be great to support all sorts of OpenCV interop, but if it's complicated just supporting mjpg is fine. I think I've seen other people do this elsewhere, and it'd be nice to have a single source for this.